index.html

```diff
@@ -4259,6 +4239,98 @@ <h2>
+          which the method was called.
+          </li>
+          <li>If <var>request</var>.<a>[[\state]]</a> is not "<a>created</a>",
+          then return <a>a promise rejected with</a> an
+          "<a>InvalidStateError</a>" <a>DOMException</a>.
+          </li>
+          <li data-tests=
+          "payment-request-hasenrolledinstrument-method-protection.https.html, payment-request-canmakepayment-method-protection.https.html">
+          Optionally, at the <a>top-level browsing context</a>'s discretion,
+          return <a>a promise rejected with</a> a "<a>NotAllowedError</a>"
+          <a> DOMException</a>.
+            <p class="note" data-link-for="PaymentRequest">
+              This allows user agents to apply heuristics to detect and prevent
+              abuse of the calling method for fingerprinting purposes, such as
+              creating <a>PaymentRequest</a> objects with a variety of supported
+              <a>payment methods</a> and triggering the <a>can make payment
+              algorithm</a> on them one after the other. For example, a user
+              agent may restrict the number of successful calls that can be made
+              based on the <a>top-level browsing context</a> or the time period
+              in which those calls were made.
+            </p>
+          </li>
+          <li>Let <var>promise</var> be <a>a new promise</a>.
+          </li>
+          <li>Return <var>promise</var>, and perform the remaining
+          steps <a>in parallel</a>.
+          </li>
+          <li>For each <var>paymentMethod</var> tuple in
+          <var>request</var>.<a>[[\serializedMethodData]]</a>:
+            <ol>
+              <li>Let <var>identifier</var> be the first element in the
+              <var>paymentMethod</var> tuple.
+              </li>
+              <li>If <var>checkForInstruments</var> is false, and the user agent
+                has a <a>payment handler</a> that supports handling payment
+                requests for <var>identifier</var>, resolve <var>promise</var>
+                with true and terminate this algorithm.
+              </li>
+              <li>If <var>checkForInstruments</var> is true:
+                <ol>
+                  <li>Let <var>data</var> be the result of <a data-cite=
+                  "ECMASCRIPT#sec-json.parse">JSON-parsing</a> the second element
+                  in the <var>paymentMethod</var> tuple.
+                  </li>
+                  <li>If required by the specification that defines the
+                  <var>identifier</var>, then <a data-cite=
+                  "WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a>
+                  <var>data</var> to an IDL value. Otherwise, <a data-cite=
+                  "WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a> to
+                  <a data-cite="WEBIDL#idl-object">object</a>.
+                  </li>
+                  <li>If conversion results in an <a data-cite=
```

Comment:
I think we can drop this `li`, because we did this check in the constructor so this can't ever throw.

Reply:
Done.